src/panels/config/zwave/zwave-node-protection.js

```diff
@@ -0,0 +1,165 @@
+import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-listbox/paper-listbox.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../../../components/buttons/ha-call-api-button.js';
+
+class ZwaveProtectionConfig extends PolymerElement {
+  static get template() {
+    return html`
+    <style include="iron-flex ha-style">
+      .card-actions.warning ha-call-api-button {
+        color: var(--google-red-500);
+      }
+      .content {
+        margin-top: 24px;
+      }
+
+      paper-card {
+        display: block;
+        margin: 0 auto;
+        max-width: 600px;
+      }
+
+      .device-picker {
+        @apply --layout-horizontal;
+        @apply --layout-center-center;
+        padding-left: 24px;
+        padding-right: 24px;
+        padding-bottom: 24px;
+        }
+
+    </style>
+      <div class="content">
+        <paper-card heading="Node protection">
+          <div class="device-picker">
+          <paper-dropdown-menu label="Protection" dynamic-align="" class="flex" placeholder="{{loadedProtectionValue}}">
+            <paper-listbox slot="dropdown-content" selected="{{selectedProtectionParameter}}">
+              <template is="dom-repeat" items="[[protectionOptions]]" as="state">
+                <paper-item>[[state]]</paper-item>
+              </template>
+            </paper-listbox>
+          </paper-dropdown-menu>
+          </div>
+          <div class="card-actions">
+            <ha-call-api-button hass="[[hass]]" path="[[nodePath]]" data="[[protectionData]]">Set Protection</ha-call-service-button>
+          </div>
+        </div>
+`;
+  }
+
+  static get properties() {
+    return {
+      hass: {
+        type: Object,
+      },
+
+      nodes: {
+        type: Array,
+        observer: 'nodesChanged'
+      },
+
+      selectedNode: {
+        type: Number,
+        value: -1,
+        observer: 'nodesChanged'
+      },
+
+      protectionNode: {
+        type: Boolean,
+        value: false,
+      },
+
+      protectionValueID: {
```

Comment:
This and all properties below it are internal and should be prefixed with an `_`